guides/05_custom-components/05_frontend.md

```diff
@@ -270,6 +270,15 @@ Svelte options:
 
 The `gradio.config.js` file should be placed in the root of your component's `frontend` directory. A default config file is created for you when you create a new component. But you can also create your own config file and use it to customize your component's build process.
 
+### Example for TailwindCSS
```

Comment:
Maybe we could name this section 'Using vite plugins' and use tailwind as the example, just so it is clear other vite plugins work.